srcpkgs/GPaste/template

```diff
@@ -1,23 +1,25 @@
 # Template file for 'GPaste'
 pkgname=GPaste
-version=42.1
+version=43.0
 revision=1
 build_helper="gir"
 build_style=meson
 configure_args="$(vopt_bool gnome_shell gnome-shell)
  -Dcontrol-center-keybindings-dir=/usr/share/gnome-control-center/keybindings
  -Dsystemd-user-unit-dir=/usr/lib/systemd"
```

Comment:
This should probably be `/usr/lib/systemd/user`

Reply:
thanks, done in 14a20ebadfe5e4db3c7f913c07ce20f44f3b1b37